python/cuml/ensemble/randomforest_shared.pxd

```diff
@@ -45,6 +45,32 @@ cdef extern from "treelite/c_api.h":
                                          ModelHandle model)
     cdef const char* TreeliteGetLastError()
 
+cdef extern from "cuml/fil/fil.h" namespace "ML::fil":
+    cdef enum algo_t:
```

Comment:
These types exist in the FIL Python wrapper already.

Reply:
I was testing the ModelHandles and trying to see if it would be better to call the C++ fil function from the rf file or use the fil.pyx file. This will be removed in the final version, as this is at the moment excess code.